pkg/api/v1/types.go

```diff
@@ -944,6 +946,30 @@ const (
 	SecretVolumeSourceDefaultMode int32 = 0644
 )
 
+// Adapts a secret into a projected volume.
+//
+// The contents of the target Secret's Data field will be presented in a
+// projected volume as files using the keys in the Data field as the file names.
+// Secret projection volumes support ownership management and SELinux relabeling.
```

Comment:
This part of the comment re: secret projection volumes support .... should only be on the ProjectedVolumeSource, not on the projections.